lib/bigshift/cli.rb

```diff
@@ -212,6 +215,8 @@ def rs_connection
         password: @config[:rs_credentials]['password'],
         sslmode: 'require'
       )
+      @rs_connection.exec("SET search_path = \"#{@config[:rs_schema_name]}\"")
```

Comment:
@iconara doesn't work (I've tried) ... that's because placeholders are for values and not identifiers which the schema name counts as.

See: http://stackoverflow.com/questions/25839259/pg-exec-params-not-replacing-parameters